lib/pf9-builder/testcase-03-centos.csv

```diff
@@ -0,0 +1,10 @@
+## kubernetes
```

Comment:
Please remove these files from the pull request, we should ultimately distribute internally and add to the gitignore